packages/web/src/common/store/pages/search-page/lineups/tracks/sagas.ts

```diff
@@ -37,38 +35,20 @@ function* getSearchPageResultsTracks({
     isMobileWeb() ||
     category === SearchKind.ALL
   ) {
-    // If we are on the tracks sub-page of search or mobile, which we should paginate on
-    let results: Track[]
-    if (isTagSearch) {
-      const { tracks } = yield* call(
-        getTagSearchResults,
-        trimToAlphaNumeric(query),
+    const currentUserId = yield* select(getUserId)
+    // searchApiFetch.getSearchResults handles tag search automatically based on query,
+    const { tracks }: { tracks: Track[] } = yield* call(
```

Comment:
TODO for me: Make sure this works correctly (i.e. components calling this don't pre-strip the hashtag)